gent_base.theme

```diff
@@ -423,7 +423,7 @@ function gent_base_preprocess_node(&$variables) {
 
   $renderChanged = [
     '#markup' => t('Last changed: @date', [
-      '@date' => \Drupal::service('date.formatter')->format($changed, 'custom', 'l j F Y - H:i '),
+      '@date' => \Drupal::service('date.formatter')->format($changed, 'custom', 'l j F Y - H.i '),
```

Comment:
Lijkt me niet volledig correct. Ik heb een opmerking geplaatst in Analyse https://digipolisgent.atlassian.net/wiki/spaces/SG8/pages/501809527/Huisstijl+-+Taalgebruik
Misschien moet je dit ticket even op blocked zetten :)

Reply:
Kunnen we die PR annuleren? Is verwarrend.